robottelo/config/validators.py

```diff
@@ -1,10 +1,75 @@
 from dynaconf import Validator
 
+from robottelo.config.vault import secrets
 from robottelo.constants import AZURERM_VALID_REGIONS
 from robottelo.constants import VALID_GCE_ZONES
 
 
 VALIDATORS = dict(
+    # Loading secrets from vault at the top,
+    # so that bulk validation fail in some setting wont skip further secrets loading
```

Comment:
How would this happen? Wouldn't it only fail for that certain section?

Reply:
validation currently happens on the entire settings object and once a validation error is encountered, it stops